# Request
---
# Result
geometry_msgs/Pose pose
---
# Feedback